### PR TITLE
chore: Remove the description of drag in the pipeline

### LINKED ISSUE #1930: Cannot drag and drop tasks to sort in pipeline editor
**Describe the bug**
I cannot drag and drop tasks to sort in pipeline editor.

image

**Versions used(KubeSphere/Kubernetes)**
KubeSphere: v3.1.0

**To Reproduce**
1. Go to 'Pipeline Editor'
2. Create some tasks
3. Try to drag and drop tasks to sort
4. No effect

**Expected behavior**
Make tasks sortable.

/kind bug
/area devops
/priority low

### PR DESCRIPTION
**Pull request by harrisonliu5**
Signed-off-by: harrisonliu5 <harrisonliu_5@163.com>

**What type of PR is this?**

 /kind optimization

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1930

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

/assign @JohnNiang

### CODE REVIEW

**Comment by JohnNiang** [comment]
@harrisonliu5 Thanks for your pr. But what I really expect is to make tasks drag and drop instead of remove the description of drag in the pipeline directly. This function is helpful if users want to sort their tasks as they want.

**Comment by harrisonliu5** [comment]
This feature was available in the previous version, because when there are a lot of task steps and stages, the drag-and-drop function will cause performance problems, so it was removed.

**Comment by JohnNiang** [comment]
> This feature was available in the previous version, because when there are a lot of task steps and stages, the drag-and-drop function will cause performance problems, so it was removed.

Got it.

File: src/pages/devops/components/Pipeline/StepsEditor/index.jsx [hunk 1]
```diff
@@ -492,10 +492,7 @@ export default class StepsEditor extends React.Component {
               {t('Add conditions')}
             </div>
           )}
-          <div className={styles.title}>
-            {t('Task')}
-            <span>{t('Drag and drop tasks to sort')}</span>
```
Comment (JohnNiang) [inline src/pages/devops/components/Pipeline/StepsEditor/index.jsx:497]: Maybe you can delete the corresponding locale as well.

Comment (LinuxSuRen) [reply src/pages/devops/components/Pipeline/StepsEditor/index.jsx:497]: I prefer to keep them for now. The feature `Drag and Drop` is excellent. I hope we can have it in the future.

Comment (JohnNiang) [reply src/pages/devops/components/Pipeline/StepsEditor/index.jsx:497]: Alright.

**Pull request merged**